common/constants/custom_panels.ts

```diff
@@ -11,3 +13,79 @@ export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of thi
 export const CUSTOM_PANELS_SAVED_OBJECT_TYPE = 'observability-panel';
 
 export const CUSTOM_PANEL_SLICE = 'customPanel';
+
+export const samplePanelName = '[Logs] Web traffic Panel';
+
+export const createDemoPanel = (savedVisualizationIds: string[]) => {
+  return {
```

Comment:
I think if we're explicitly indexing the savedVisualizationIds then we should at least assert the list-length on the way in.

An alternative would be to put the list of panel-locations into its own array and zip() the savedVisualizationIds and panelLocations... then take whatever has [not-null, not-null].

Might be a lot easier to read both the locations list (could be listed horizontally), and the structure of the visualization (listed once in a map() from the zip() result).